### DISCUSSION TITLE
Welcome to quantum-mechanics Discussions!

### DISCUSSION THREAD

**Discussion by dou-du** [osscar-org/quantum-mechanics#325] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Laurentina-Hill** [comment]
Hello, is there any chance I could see the code you used to graph your phonon bands for the honeycomb lattice example you have? I have been trying to recreate that image for my own understanding towards my thesis and have been unable to do so as of yet. I am hoping that seeing an accurate Dynamic matrix may give some insight into where my calculations are going wrong.
